src/Servers/testassets/TestSites/ServerComparison.TestSites.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
-    <NativeProjectReference Include="$(MSBuildThisFileDirectory)src\AspNetCoreModuleV1\AspNetCore\AspNetCore.vcxproj" />
+    <NativeProjectReference Include="$(RepositoryRoot)src\AspNetCoreModuleV1\AspNetCore\AspNetCore.vcxproj" />
```

Comment:
Can these be simplified to this?

```suggestion
    <Reference Include="AspNetCoreModuleV1" />
```

Reply:
To be honest, I don't think `Reference Include="AspNetCoreModule"` has ever worked. I try to include it and and it never get's removed and replaced with the NativeProjectReference. It fails on restore.

Reply:
I didn't see this helping either.